app/services/blacklight/field_retriever.rb

```diff
@@ -22,17 +22,19 @@ def initialize(document, field_config, view_context = nil)
 
     # @return [Array]
     def fetch
-      Array.wrap(
-        if field_config.highlight
-          retrieve_highlight
-        elsif field_config.accessor
-          retieve_using_accessor
-        elsif field_config.values
-          retrieve_values
-        else
-          retrieve_simple
-        end
-      )
+      if field_config.highlight
```

Comment:
It's funny, I almost wish that the `retrieve_` methods just reutnred nil if `accessor` or `values` or whatever wasn't set, so we could do `retrieve_highlight || retrieve_using_accessor || retrieve_values || retrieve_simple`, but that's probably too big for here.